### PR TITLE
Add FinderMinder v1.2

### PR DESCRIPTION
**Pull request by danielbayley**
FinderMinder by Irradiated Software

Signed-off-by: Daniel Bayley daniel.bayley@me.com

### CODE REVIEW

File: Casks/finderminder.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+cask :v1 => 'finderminder' do
+  version :latest
+  sha256 :nocheck
```
Comment (vitorgalvao) [inline Casks/finderminder.rb]: `:no_check`.

**Comment by danielbayley** [comment]
There's an option to squash commits in Tower, but I think it can only be done when merging one branch into another. The homebrew-cask guidelines say to submit PRs from the new branch, so what to do?

**Comment by vitorgalvao** [comment]
Whatever you did, was correct.

**Comment by danielbayley** [comment]
@vitorgalvao Strangely i'm getting an `Uh oh, could not identify primary container for '/Library/Caches/Homebrew/finderminder-latest.zip'` error when doing `brew install finderminder`. What could be causing this?

**Comment by vitorgalvao** [comment]
`url` is wrong. It’s pointing to an html page. Correct link is `http://www.irradiatedsoftware.com/download/FinderMinder.zip`. Care to fix it?

**Comment by danielbayley** [comment]
Sure.

> don’t forget to confirm the checksum

I used `:no_check` because the link doesn't contain a version number, and according to the docs;
```
This combination of version :latest and sha256 :no_check is currently the preferred mechanism when an unversioned download URL is available
```
I therefore shouldn't right?

**Comment by vitorgalvao** [comment]
You can’t fix it here since it has been merged; it needs to be a new PR.

Oh, right, I forgot this was a `:no_check`. In that case, only `url` needs changing. Don’t worry about it, I can do it faster, since I can commit directly from the web interface.

**Comment by danielbayley** [comment]
Already figured it out I think. See #11698

**Pull request merged**